docs/UserGuide.adoc

```diff
@@ -263,6 +263,13 @@ Exits the program. +
 Format: `exit` +
 Format: `x`
 
+=== Keyboard shortcuts [Since v1.2]
```

Comment:
Do we need the [Since v1.2]?

Reply:
Should be. You see the v1.2 under documentation for user guide